test/llvm_autogenerated/llvm-to-s.py

```diff
@@ -47,18 +44,25 @@ def GetRunLine(test):
 
 def main(args):
     parser = argparse.ArgumentParser()
+    parser.add_argument('-l', '--llvm-dir', required=True)
```

Comment:
Would `required=False` with a default of `os.path.join(ROOT_DIR, 'third_party', ...` make sense?